clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -9924,7 +9924,11 @@ void OffloadBundler::ConstructJob(Compilation &C, const JobAction &JA,
       T.setArchName(ArchName);
       Triples += T.normalize();
     } else {
-      Triples += CurTC->getTriple().normalize(llvm::Triple::CanonicalForm::FOUR_IDENT);
+      if (CurKind == Action::OFK_HIP)
```

Comment:
Can we please add a comment here? May be a TODO is needed here? Thanks

Reply:
Added here https://github.com/intel/llvm/pull/17557/commits/7ff5305a4cff4088e3284ed0fb09b91a2a02ff44 is it sufficient?